macros/SpiderMonkeySidebar.ejs

```diff
@@ -49,62 +49,86 @@ var text = mdn.localStringMap({
   <li><a href="/<%=locale%>/docs/Mozilla/Projects/SpiderMonkey"><strong><em>SpiderMonkey</em></strong></a></li>
   
   <li><strong><%=text['References']%></strong></li>
-  <li data-default-state="<%=state('JSAPI')%>"><a href="/<%=locale%>/docs/Mozilla/Projects/SpiderMonkey/JSAPI_reference"><%=text['JSAPI_reference']%></a>
-    <%-template("ListSubpagesForSidebar", ['/' + locale + '/docs/Mozilla/Projects/SpiderMonkey/JSAPI_reference'])%>
+  <li class="toggle">
```

Comment:
The ``{{SpiderMonkeySidebar}}`` macro is converted correctly, with the exception of the "Release Notes" text element, below the fold on https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/JSAPI_reference/JS::AutoValueArray .

<img width="567" alt="pr 789 spidermonkeysidebar" src="https://user-images.githubusercontent.com/286017/45394836-683ac900-b5f8-11e8-9c40-0fcd44f82aac.png">